### DISCUSSION TITLE
Logging in just sends a request to /api/auth/signin?csrf=true and errors

### DISCUSSION THREAD

**Discussion by Misly16** [nextauthjs/next-auth#2528] [Help]
### Question 💬

Hi, whenever trying to login it just sends a request to /api/auth/signin?crsf=true and then I get a json parse error. This worked fine previously and I haven't made any changes so I'm not sure why it isnt working anymore.

My NEXTAUTH_URL variable is set to `http://localhost:3000/api/auth`

### How to reproduce ☕️

Click signin
![image](https://user-images.githubusercontent.com/60405462/129446628-55293434-9779-42f4-ba58-02c3c1609588.png)

### Contributing 🙌🏽

No, I am afraid I cannot help regarding this

**Comment by prasanthbattula** [comment]
I got the same issue.
I resolved the issue by adding NEXTAUTH_URL in the env variables. The url should not be your local url it should be your production url.

**Reply by Misly16** [reply]
Hi, turns out the issue was the fact that I had a custom express server infront of the application

**Reply by Misly16** [reply]
Not sure how I'm supposed to get it working with it though
